pkg/cli/alpha/config-gen/testdata/enablecertmanager/expected.yaml

```diff
@@ -353,7 +353,7 @@ webhooks:
 ---
 # The following manifests contain a self-signed issuer CR and a certificate CR.
 # More document can be found at https://docs.cert-manager.io
-# WARNING: Targets CertManager 0.11 check https://docs.cert-manager.io/en/latest/tasks/upgrading/index.html for 
+# WARNING: Targets CertManager 0.11 check https://docs.cert-manager.io/en/latest/tasks/upgrading/index.html for
```

Comment:
Remove the space here broke the tests. 
See that it is a mock test result so that we can compare what is the result with it (what is expected)
You can check that by looking in the logs: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kubebuilder/2595/pull-kubebuilder-test/1510670366261907456#1:build-log.txt%3A77-84

c/c @NikhilSharmaWe 

Revert this one for we are able to move forward.

Reply:
I saw that in the logs but did not realize that it was important for them to be exactly same. 
Thanks